### DISCUSSION TITLE
Bug Report

### DISCUSSION THREAD

**Discussion by Skyfly2020** [FakeFishGames/Barotrauma#15315] [Bug reports]
### Disclaimers

- [x] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

the day before yesterday I went on a mission and got stuck between rocks, then uploaded the last save where I just sailed and returned to base, and then decided to sail in a different direction. Today I returned to the game and decided to swim in the direction I was stuck in, but I chose other tasks and clicked "Swim". The problem is that I was teleported immediately to the place where I was stuck between the rocks, and not to the beginning of the location. The mistake is probably that when I got stuck, the game saved this part of the mission in memory. And when I returned to this mission again, I was rolled back to this save at the moment where I was stuck. 

While I was writing the whole text, I was already able to swim out of the rocks, but still I don't know the reason for the bug

![2024-12-08_00-48-16](https://github.com/user-attachments/assets/bb548fc3-548c-4a4d-bb27-ac3bc19dcd75)

### Reproduction steps

1. Get stuck in the rocks
2. Roll back to the last save
3. Sail on another mission
4. Return back to the same base
5. Repeat the mission you are stuck on

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Folowed your steps and couldn't reproduce

>  the game saved this part of the mission in memory

no, it can't do that

> Bug prevalence
Just once

That must've been a very special save file, that is now lost forever and we'll never know
Unless you stashed it somewhere and is able to zip it and put it here
